docs/gitlab/tests.md

```diff
@@ -0,0 +1,15 @@
+## Unity tests
```

Comment:
Tests

Reply:
I think I'll duplicate the titles here, because the text bellow is about unity tests, but then it's about what we do with them 👍